$NetBSD: patch-ac,v 1.3 2015/12/29 23:34:43 dholland Exp $

Avoid symbol name conflict with random().
XXX: should not be using the implementation namespace.

--- interp/random.c.orig	1999-08-31 18:19:18.000000000 +0200
+++ interp/random.c
@@ -65,7 +65,7 @@ long int temp;
    useed = temp+seed;   /* start user somewhere else */
 }
 
-unsigned int   random(range)
+unsigned int   __random(range)
 unsigned int   range;
    /* returns a pseudo-random unsigned int between 0 and 'range' - 1 */
 {
